src/Umbraco.Web.UI.NetCore/appsettings.json

```diff
@@ -38,8 +38,8 @@
         "ConvertUrlsToAscii": "try"
       },
       "RuntimeMinification": {
-        "dataFolder": "App_Data/TEMP/Smidge",
-        "version": "637432008251409860"
+        "dataFolder": "Umbraco/TEMP/Smidge",
```

Comment:
Is moving this to under Umbraco intentional?

Reply:
Yes, but actually this should be `Umbraco/Data/TEMP/Smidge`. App_Data is not a thing in netcore. Each folder outside of www is essentially a form of 'app data' :) I'll update the path.